include/seqan3/io/sequence_file/input_format_concept.hpp

```diff
@@ -38,13 +38,13 @@ namespace seqan3
  */
 //!\cond
 template <typename t>
-SEQAN3_CONCEPT sequence_file_input_format_concept = requires (t                                     & v,
-                                                         std::ifstream                         & f,
-                                                         sequence_file_input_options<dna5, false> & options,
-                                                         dna5_vector                           & seq,
-                                                         std::string                           & id,
-                                                         std::vector<phred42>                  & qual,
-                                                         std::vector<dna5q>                    & seq_qual)
+SEQAN3_CONCEPT SequenceFileInputFormat = requires (t                                     & v,
+                                                   std::ifstream                         & f,
+                                                   sequence_file_input_options<dna5, false> & options,
+                                                   dna5_vector                           & seq,
+                                                   std::string                           & id,
+                                                   std::vector<phred42>                  & qual,
+                                                   std::vector<dna5q>                    & seq_qual)
```

Comment:
If you already touch all of these line, can you align the `&` to be in one column? That was the original plan!